src/DistributedLock.Tests/Infrastructure/Azure/AzureSetUpFixture.cs

```diff
@@ -8,60 +9,36 @@ namespace Medallion.Threading.Tests.Azure;
 [SetUpFixture]
 public class AzureSetUpFixture
 {
-    // https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio%2Cblob-storage
-    private const string EmulatorProcessName = "azurite";
-
-    private bool _startedEmulator;
```

Comment:
Why are these changes to AzureSetUpFixture necessary? If they aren't let's revert them since they seem unrelated to this work